psqlextra/compiler.py

```diff
@@ -187,12 +187,13 @@ def execute_sql(self, return_id=False):
                     rows.extend(cursor.fetchall())
                 except ProgrammingError:
                     pass
+            description = cursor.description
 
         # create a mapping between column names and column value
         return [
             {
                 column.name: row[column_index]
-                for column_index, column in enumerate(cursor.description)
+                for column_index, column in enumerate(description)
```

Comment:
I don't understand this change. How is aliasing it on line 190 changing how this works?

Reply:
It seems that the `with cursor` now cleans up the data on the cursor on exit. So `cusror.description` ends up being None. So we just need to read it before the with block exits